aten/src/ATen/native/quantized/QTensor.cpp

```diff
@@ -11,6 +11,43 @@
 namespace at {
 namespace native {
 
+Tensor quantize_per_tensor_dynamic(
+    const Tensor& self,
+    ScalarType dtype,
+    bool reduce_range) {
+  TORCH_CHECK( (dtype == ScalarType::QInt8 || dtype == ScalarType::QUInt8), "dtype ", dtype, "not supported");
+  auto input_contig = self.contiguous();
+  float x_min = input_contig.min().item<float>();
+  float x_max = input_contig.max().item<float>();
+
+  if (reduce_range && at::globalContext().qEngine() == at::QEngine::QNNPACK) {
+    reduce_range = false;
+  }
+
+  int qmin = 0;
+  int qmax = 255;
```

Comment:
nit: maybe remove the initializations since qmin and qmax are determined by dtype

Reply:
I'll make the change. For future reference, is the practice at FB / PyTorch to leave variables uninitialized if you know that it'll be set later (before it's used)? I'm used to initializing everything :)

Reply:
I don't remember if there is a practice like that, I feel this should be OK. There are also cases when it does make sense to initialize like: https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/native/quantized/cpu/q_avgpool.cpp#L406, so I think leave things uninitialized can happen sometimes.

For this particular case, since the choice of the initialized value is a bit arbitrary I feel it is clearer just to leave it uninitialized.